.changelog/10304.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: Adding a notice about task ARN format to Amazon ECS docs
+```
```

Comment:
Sorry, you don't need this for doc PRs. I can add the pr:no-changelog label to fix the failing check.